### PR TITLE
Adds the first of a series of Github Challenges.

### PR DESCRIPTION
**Pull request by MaineC**
As discussed here:

https://issues.apache.org/jira/browse/TRAINING-5

this adds the first of a series of challenges. As those are prepared on a weekly basis, I'll likely post more.

### CODE REVIEW

**Comment by justinmclean** [comment]
Look fine to me

File: content/Github/Challenges/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+-->
+
+# Github
```
Comment (lfrancke) [inline content/Github/Challenges/README.md]:
```suggestion
# GitHub
```

File: content/Github/Challenges/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Several projects at the Apache Software Foundation are using Git and Github for
```
Comment (lfrancke) [inline content/Github/Challenges/README.md]:
```suggestion
Several projects at the Apache Software Foundation are using Git and GitHub for
```

File: content/Github/Challenges/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+their daily business. However not all potential contributors are comfortable
+using that toolset - or are aware of how deep their knowledge of that tooling
+is.
+
+This directory contains small challenges that can help gauge interest in
+learning more about Git and Github: One example use case would be printing the
```
Comment (lfrancke) [inline content/Github/Challenges/README.md]:
```suggestion
learning more about Git and GitHub: One example use case would be printing the
```

File: content/Github/Challenges/workflow-basics.svg [hunk 4]
```diff
@@ -0,0 +1,2495 @@
...
+         sodipodi:role="line"
+         x="197.17068"
+         y="290.96933"
+         style="line-height:1;text-align:end;text-anchor:end;stroke-width:0.26458332"
+         id="tspan4071">GitHub, the GitHub and the Octocat logo are registered trademarks of GitHub Inc.  Git and the Git logo are either registered trademarks or trademarks of Software Freedom </tspan><tspan
+         sodipodi:role="line"
+         x="196.49683"
+         y="293.086"
+         style="line-height:1;text-align:end;text-anchor:end;stroke-width:0.26458332"
+         id="tspan4081">Conservancy, Inc.,  corporate home of the Git Project, in the United States and/or other countries. Europace and the Europace logo are registered trademarks of Europace AG.</tspan></text>
```
Comment (willshen) [inline content/Github/Challenges/workflow-basics.svg]: nitpick: noticed the following at the bottom of the graphic, not sure if it is still needed.
> Europace and the Europace logo are registered trademarks of Europace AG.

**Comment by MaineC** [comment]
Added solutions in rot13 encoding, see also #41 for the original suggestion

**Comment by MaineC** [comment]
@lfrancke @justinmclean Do you need anything else before this can be merged?

**Comment by justinmclean** [comment]
I'd add a LICENSE and NOTICE file. They can be boiler plate, but I would add something about having permission to use the GitHub image. Having an ASF header on the SVG file would also be a good idea.

**Comment by MaineC** [comment]
@justinmclean added the license header and the information about explicit permission to use octocats in the material to Notice.

**Pull request merged**